### PR TITLE
XW-3180 | recirculation clean up, phantom update

### PR DESCRIPTION
**Pull request by latata**
https://wikia-inc.atlassian.net/browse/XW-3180

@Wikia/x-wing

### CODE REVIEW

**Review by rybmat — APPROVED** [review]

**Review by weronika — APPROVED** [review]

File: extensions/wikia/Recirculation/js/recirculation.js [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+						}
+
+						data.items = data.items.concat(result.items);
+					});
+
+					view.render(data)
+						.then(view.setupTracking())
+						.then(deferred.resolve);
+				})
+				.fail(handleError(viewModule), function () {
```
Comment (kvas-damian) [inline extensions/wikia/Recirculation/js/recirculation.js:146]: Is this code suppose to be an ABTest or is it production implementation?
If we're productizing this feature maybe it makes sense to render some recirc modules when only one promise is rejected?

File: extensions/wikia/Recirculation/RecirculationController.class.php [hunk 2]
```diff
@@ -14,23 +14,6 @@ public function init() {
 		}
 	}
 
-	public function index() {
```
Comment (rogatty) [inline extensions/wikia/Recirculation/RecirculationController.class.php:17]: Where was it used previously? I can't find any references.

File: extensions/wikia/Recirculation/js/recirculation.js [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+             discussions,
+             liftigniter,
+             videosModule) {
+	/**
+	 *
+	 *  var recircModules = [
+	 *    {
+	 *  		id: 'unique identifier - only used if you want to use data across multiple views',
+	 *  		viewModule: 'name of the view',
+	 *  		helper: 'name of the helper',
```
Comment (rogatty) [inline extensions/wikia/Recirculation/js/recirculation.js]: `helper` is not used anymore.

File: extensions/wikia/Recirculation/js/recirculation.js [hunk 4]
```diff
@@ -0,0 +1,184 @@
...
+	 *  		options: {
```
Comment (rogatty) [inline extensions/wikia/Recirculation/js/recirculation.js]: `opts`?

File: extensions/wikia/Recirculation/js/recirculation.js [hunk 5]
```diff
@@ -0,0 +1,184 @@
...
+	 *  			any options to pass to the helper
```
Comment (rogatty) [inline extensions/wikia/Recirculation/js/recirculation.js]: `pass to liftigniter`?

File: extensions/wikia/Recirculation/js/recirculation.js [hunk 6]
```diff
@@ -0,0 +1,184 @@
...
+	 *  		}
+	 *  	}
+	 *  ];
+	 *
+	 */
+	'use strict';
+
+	var recircModules = [
+			{
+				id: "LI_rail",
```
Comment (rogatty) [inline extensions/wikia/Recirculation/js/recirculation.js]: How about sticking to single quotes?

File: extensions/wikia/Recirculation/js/recirculation.js [hunk 7]
```diff
@@ -0,0 +1,184 @@
...
+					flush: true,
+					opts: {
+						resultType: 'subdomain',
+						domainType: 'fandom.wikia.com'
+					}
+				}
+			}
+		],
+		logGroup = 'ext.wikia.recirculation.experiments.mix',
+		views = {}, // Each view holds an array of promises used to gather data for that view
```
Comment (rogatty) [inline extensions/wikia/Recirculation/js/recirculation.js]: https://github.com/Wikia/guidelines/blob/master/JavaScript/CodingConventions.md#comments
>Warning: Use of inline comments in not allowed (with the exception of ESLint line declarations)

**Review by rogatty — APPROVED** [review]

File: extensions/wikia/Recirculation/RecirculationController.class.php (hunk 2 above)
Comment (latata) [reply extensions/wikia/Recirculation/RecirculationController.class.php:17]: https://github.com/Wikia/app/commit/19cc2df13f86afa64f5cc569920933e0ff49a13e

**Pull request merged**